r2/r2/models/promo.py

```diff
@@ -39,7 +39,9 @@
 PROMOTE_STATUS = Enum("unpaid", "unseen", "accepted", "rejected",
                       "pending", "promoted", "finished")
 
-class PriorityLevel(object):
+PROMOTE_COST_BASIS = Enum('fixed_cpm', 'cpm', 'cpc',)
+
+
     name = ''
```

Comment:
what's going on here? it looks like all these bits that used to be in the PriorityLevel class have been orphaned?